integration/src/mayachain/index.ts

```diff
@@ -0,0 +1,7 @@
+import * as core from "@shapeshiftoss/hdwallet-core";
+
+import { mayachainTests as tests } from "./mayachain";
```

Comment:
If you follow the naming convention of the integrations module, you won't have to alias the import